### PR TITLE
Use DateTime type in schema instead of Date

### LINKED ISSUE #5: graphql-scalars fails to parse incoming datetimes from FOLIO
When asking for a `Loan`'s `dueDate` (and other datetimes), we seem to get back a timestamp from FOLIO that we can't parse:

> TypeError: Date cannot represent an invalid date-string 2022-09-28T06:59:59.000+00:00.

I think this is happening under the covers based on the `Date` resolver provided by `graphql-scalars`.

### PR DESCRIPTION
**Pull request by marlo-longley**
Fixes #5

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -123,7 +123,7 @@ const resolvers = {
       return types.getMapFor<Campus>("location-units/campuses", { key: "loccamps", cache: context.typeCache }).then(map => map.get(parent.campusId))
     }
   },
-  Date: DateResolver,
+  Date: DateTimeResolver,
```
Comment (cbeer) [inline src/index.ts:126]: Would it be clearer if we made this a `DateTime` type and updated the schema to use it?

Comment (marlo-longley) [reply src/index.ts:126]: @cbeer do you mean replaced Date with DateTime all ~34 times it occurs in `schema.graphql` ? sure !

Comment (cbeer) [reply src/index.ts:126]: If they're all DateTime values, yep.

Comment (marlo-longley) [reply src/index.ts]: @cbeer OK this should be good. I checked the docs / responses in Apollo and they all seem good with DateTime.

**Pull request merged**